atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/domain/buildingblocks/Pollux.kt

```diff
@@ -125,4 +130,21 @@ interface Pollux {
     suspend fun getCredentialDefinition(id: String): CredentialDefinition
 
     suspend fun getSchema(schemaId: String): Schema
+
+    suspend fun createPresentationDefinitionRequest(
+        type: CredentialType,
+        fromDID: DID,
+        toDID: DID,
+        proofs: Array<ProofTypes>,
+        options: PresentationOptions = PresentationOptions()
+    ): PresentationDefinition
```

Comment:
minor point, but the function name and return type are mismatched: is this for a PresentationDefinitionRequest or a PresentationDefinition?

Reply:
Good point